tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj

```diff
@@ -49,7 +49,7 @@
     <PackageReference Include="Microsoft.Windows.SDK.Contracts" Version="10.0.19041.1" />
     <ProjectReference Include="..\..\src\client\Microsoft.Identity.Client.Desktop\Microsoft.Identity.Client.Desktop.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' ">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3' ">
```

Comment:
Why does this one need to change?
The unit tests only use net core and net framework?

Reply:
This is just for `Microsoft.Identity.Client.Broker` as it does not have a target for `netcoreapp3.1`

Reply:
Does it have to be netstandard2.0 then or is this based on MSAL's netstandard1.3?

Reply:
this is based on `<TargetFrameworkNetStandard>netstandard1.3</TargetFrameworkNetStandard>` in `Microsoft.Identity.Client`

Reply:
This doesn't look right. Our tests run against: 

    <TargetFrameworkNetDesktop461>net472</TargetFrameworkNetDesktop461>
    <TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
    <TargetFrameworkNet5Win>net5.0-windows10.0.17763.0</TargetFrameworkNet5Win>

None of them run against `'netstandard1.3'` so this item group would simply be ignored.